lib/reader.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require './modules/validation'
+class Reader
+  include Validation
+  attr_accessor :name, :email, :city, :street, :house
+
+  def initialize(name, email, city, street, house)
```

Comment:
Let's reduce the number of args.
Pass `address` as, for example, `Hash` object then extract the needed data and assign to variables.
[refactoring slides](https://rubygarage.github.io/slides/refactoring-ruby#/18)